src/site/content/en/blog/web-otp/index.md

```diff
@@ -196,6 +196,11 @@ formatted message text sent via SMS. I'll cover the JavaScript first.
 The Web OTP API requires a secure origin (HTTPS).
 {% endAside %}
 
+{% Aside 'success' %}
+Unless you'd like a highly customised approach, we recommend to use [a web
+component](#web-components) approach since it's mostly just a copy and paste. 
```

Comment:
```suggestion
Unless you want to highly customize the experience, [a web
component](#web-components) approach is recommended because it's mostly just a
copy and paste.
```